### PR TITLE
typos & PATH information

### PR DESCRIPTION
**Pull request by carlmontanari**
Minor clean ups for types/wording

Added note about adding packer/terraform to PATH

Thanks for a great tutorial, just a few minor tweaks!

### CODE REVIEW

File: docs/04-packer.md [hunk 1]
```diff
@@ -22,6 +22,20 @@ Check the version to verify that it was installed:
...
 ```
 
+Note: If you see an error similar to the following you may need to add Packer to your PATH:
+
+```bash
+$ packer -v
+-bash: packer: command not found
+```
+
+To add Packer to your PATH you will need to modify your shell profile to include the Packer UNIX executable file. In MacOS you can edit the `~/.bash_profile` file to include Packer by adding the following (your system may vary, this example was used on MacOS 10.13):
```
Comment (Artemmkin) [inline docs/04-packer.md]: this packer section is too specific about a particular OS and shell, and I didn't want to make any assumptions about those things. I would rather leave this section out of the tutorial and let people figure out how to install packer on their local machine. It's very straightforward, in my opinion, and the tool's documentation is sufficient to deal with this problem.

File: docs/05-terraform.md [hunk 2]
```diff
@@ -46,6 +46,8 @@ Make sure Terraform version is  => 0.11.0:
 $ terraform -v
 ```
 
+Note: If you see an error here, you may need to add Terraform to your PATH as we did with Packer.
```
Comment (Artemmkin) [inline docs/05-terraform.md]: would rather leave this out, see comments on packer

File: docs/05-terraform.md [hunk 3]
```diff
@@ -91,7 +93,7 @@ This means that Terraform has a pluggable architecture and the pluggable compone
 
 So before we can actually create a VM using Terraform, we need to define a configuration of a [google cloud provider](https://www.terraform.io/docs/providers/google/index.html) and download it on our system.
 
-Create another file inside `terraform` folder and call it `providers.tf`. Put provider configuration in it:
+Create another file inside `terraform` folder and call it `providers.tf`. Put provider configuration in it (make sure you are setting the region as appropriate):
```
Comment (Artemmkin) [inline docs/05-terraform.md]: don't quite understand  `setting the region as appropriate` part, it looks kind of confusing to me. I would rather put something like `Note the region value, this is where terraform will provision resources (you may wish to change it).` after the the code block

File: docs/07-vagrant.md [hunk 4]
```diff
@@ -152,6 +152,13 @@ end
 
 In the previous lab, it was given to you as a task to create a `configuration.yml` playbook that provides the same functionality as `configuration.sh` script we had used before. If you did not do that, you can copy the playbook from [here](https://github.com/Artemmkin/infrastructure-as-code-example/blob/master/ansible/configuration.yml) (place it inside `ansible` directory). If you did create your own playbook, make sure you have a `pre_tasks` section as in [this example](https://github.com/Artemmkin/infrastructure-as-code-example/blob/master/ansible/configuration.yml).
 
+If you see a failure similar to the one below while Vagrant is running the Ansible playbook, ensure you copy the configuration.yaml file as linked above -- if you do not the VM Vagrant creates will not have Python 2.7 installed which will cause Ansible to fail (this is what the `pre_tasks` section is for!).
```
Comment (Artemmkin) [inline docs/07-vagrant.md]: I would leave this out of the tutorial as well. I know I don't explain what `pre_tasks` are for, but for those who are truly interested it wouldn't be a problem to figure out :)

File: docs/10-kubernetes.md [hunk 5]
```diff
@@ -54,6 +54,12 @@ $ unzip gke-terraform.zip -d kubernetes/terraform
 $ rm gke-terraform.zip
 ```
 
+Note: If you are using MacOS High Sierra or newer you may have to install wget, you can do so via Homebrew (https://stackoverflow.com/questions/33886917/how-to-install-wget-in-macos-capitan-sierra/43291137).
```
Comment (Artemmkin) [inline docs/10-kubernetes.md]: again, too specific about a particular platform, I would rather leave this out

**Review by Artemmkin — COMMENTED** [review]
@carlniger hey! thanks for your contribution and feedback, I appreciate it very much! 

Grammar corrections are awesome, the tutorial needs it, since I'm not a native speaker and tend to make mistakes in wording every now and then. 

There are a few sections in your PR that I would rather not to include in the tutorial. I left comments on those. Basically, I wouldn't want to make any assumptions about the OS or shell a person uses. This tutorial can be done easily on OSX and Linux and almost all of the labs can be done on Windows as well. Every local environment is different and I believe it's up to the person who uses it to solve problems like installing the tools (and the tools' documentation is sufficient to figure it out). 

Please, leave your comments on my comments, and if we agree on the changes in question, I would be more than happy to merge your PR. 

Thanks again for taking the time and contributing!

File: docs/05-terraform.md (hunk 3 above)
Comment (carlmontanari) [reply docs/05-terraform.md]: Makes sense, happy with it as is, or with what you replied in the comment!

File: docs/07-vagrant.md (hunk 4 above)
Comment (carlmontanari) [reply docs/07-vagrant.md]: :) Took me a bit to figure out, was a good problem!

**Comment by carlmontanari** [comment]
All sounds good to me sir! 👍

**Comment by Artemmkin** [comment]
awesome! could you then delete the sections I commented on and make another commit to this pull request so I can merge it?

**Comment by carlmontanari** [comment]
Done, think I got it all covered!

**Review by Artemmkin — APPROVED** [review]
LGTM 👍

**Comment by Artemmkin** [comment]
@carlniger thanks again for taking the time and contributing!

**Pull request merged**